Symfony/CS/Tests/Config/ConfigTest.php

```diff
@@ -23,7 +23,7 @@ public function testThatDefaultFinderWorksWithDirSetOnConfig()
         $config->setDir(__DIR__.'/../Fixtures/FinderDirectory');
 
         $iterator = $config->getFinder()->getIterator();
-        $this->assertSame(1, count($iterator));
+        $this->assertCount(1, $iterator);
```

Comment:
Will revert, which PhpUnit version and bug you are referencing to (needs for my plugin as well)?